deployment/build-s3-dist.sh

```diff
@@ -22,20 +24,26 @@ if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
     exit 1
 fi
 
-export DIST_VERSION=$3
 export DIST_OUTPUT_BUCKET=$1
-export SOLUTION_ID=SO0030
 export SOLUTION_NAME=$2
-export SOLUTION_TRADEMARKEDNAME=$2
+export VERSION=$3
```

Comment:
Good catch. the DIST_VERSION usage was for the CLI which I left as-is and never manually tested, we should add cli tests as a story item to at least check before release to ensure none of the other changes over the last couple months have impacted it

"VERSON" was because that was the value expected in the helper code added to aws-instancescheduler.ts I have updated both to use "DIST_VERSION" to be more explicit, though I wonder if this is the correct name or not. The purpose of the variable is to refer to the version argument passed in by the internal pipeline and is (supposed to be) only used to produce the object name in the solutions deployment bucket. However, it looks like the CLI also takes this version in for some unknown purpose inside of setuptools (might just be to set the metadata tag on the package)